var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -556,6 +556,7 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
 
     # https://github.com/trilinos/Trilinos/pull/13921
     patch("16-1-0-stk-fpe-exceptions.patch", when="@=16.1.0 +stk platform=darwin")
+    patch("16-1-0-stk-size_t.patch", when="@=16.1.0 +stk")
```

Comment:
Can you add a brief comment above the `patch` directive explaining what is the effect of the patch? That will help future readers of the `package.py`

Reply:
Sure.